woocommerce/woocommerce-android
# [HACK Week] Improve Support Request form error and identity handling

Summary
==========
Refactor how the `ZendeskTicketRepository` reacts when there's no Contact identity information set by creating an error type system that is verified inside the `SupportRequestFormViewModel`, making sure that we will never attempt to submit the request without a valid identity set.

How to Test
==========
1. Inside the `ZendeskSettings` class, hard code the `isIdentitySet` property to `false` and make sure there's absolute no way possible to submit the request.

Update release notes:
